docs/content/stable/migrate/yb-voyager-cli.md

```diff
@@ -269,6 +312,57 @@ This argument is to be specified only for CSV file type.
 
 Default: false; change to true if the CSV file contains column names as a header.
 
+**Note**: Boolean flags takes arguments in the format `--flag-name=[true|false]`, and not `--flag-name [true|false]`.
+
+### --file-opts
+
+Comma-separated string options for CSV file format. The options can include the following:
+
+- `escape_char`: escape character
+
+- `quote_char`: character used to quote the values
+
+Default : double quotes (") for both escape and quote characters
+
+Note that `escape_char` and `quote_char` are only valid and required for CSV file format.
+
+Example: `--file-opts "escape_char=\",quote_char=\""` or `--file-opts 'escape_char=",quote_char="'`
+
+### --format
+
+Specifies the format of your data file with CSV or text as the supported formats.
+
+Default : CSV
+
+### --post-import-data
+
+Run this argument with [import schema](#import-schema) command to import indexes and triggers after data import is complete.
+The `--post-import-data` argument assumes that data import is already done and imports only indexes and triggers.
+
+### --oracle-db-sid
+
+Oracle System Identifier (SID) you can use while exporting data from Oracle instances.
+
+### --oracle-home
+
+Path to set `$ORACLE_HOME` environment variable. `tnsnames.ora` is found in `$ORACLE_HOME/network/admin`.
+
+### --use-orafce
+
+Enable using orafce extension in export schema.
+
+Default : true
+
+### --yes
+
+By default, answer yes to all questions during migration.
+
+### --start-clean
+
+Clean the project's data directory for already existing files before starting migration.
+
```

Comment:
Please also add the documentation for `--table-list` and `--exclude-table-list` in this section, I believe it has been missed in this iteration of the PR. Picked up straight from the [google doc](https://docs.google.com/document/d/1dZ7RTzKAhMgs5W9OR4NvwjtZW2M7fh3QMGpfBvQCYYM/edit#):
```
--table-list (string)             Comma separated list of the tables to for which data is exported
--exclude-table-list (string)     Comma separated list of tables to exclude while exporting data(no-op if --table-list is used)
```